_posts/2022-04-26-omero-5-6-4.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: Release of OMERO.server 5.6.4
+---
+
+Today we are releasing OMERO.server 5.6.4 which uses OMERO.blitz 5.5.10
+and includes Bio-Formats 6.9.1, see [release announcement]({{ site.baseurl }}/2022/04/20/bio-formats-6-9-1.html).
+The Bio-Formats Memoizer cache **will be invalidated** on upgrade from OMERO.server 5.6.3.
+
+The OMERO pyramid generation process should be considered as deprecated, please read the [recommendations](https://docs.openmicroscopy.org/omero/latest/sysadmins/limitations.html) on how to proceed.
```

Comment:
I was unsure of whether the limitation section would include several alternatives  but this seems to be the current consensus - see https://github.com/ome/omero-documentation/blob/f850d4c86bdc1fdbde2fc5cf24e4cfa47a51bf72/omero/sysadmins/limitations.rst#large-images so :+1: from my side.

Maybe worth linking to the anchor?